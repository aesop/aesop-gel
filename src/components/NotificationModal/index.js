/* istanbul ignore file */
import { NotificationModal } from './NotificationModal';

export { NotificationModal };
