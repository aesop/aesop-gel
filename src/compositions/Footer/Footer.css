@import 'styles/index';

.base {
  position: relative;
  background-color: var(--color-grey-10);
}

.block {
  display: flex;
  flex-wrap: wrap;
  border-bottom: 3px solid #d6d5cb;
}

.bottom {
  display: flex;
  height: 72px;
  align-items: center;
  padding-left: 25px;
  color: var(--color-light-copy);
  font-family: var(--font-family-sans-serif-regular);
}

.signup {
  flex: 1 0 30%;
  padding-top: 45px;
  padding-right: 25px;
  padding-bottom: 25px;
  padding-left: 25px;
  @media (--viewport-sm-only) {
    flex: 100%;
  }
  @media (--viewport-md-only) {
    flex: 80%;
  }
}

.item {
  flex: 1 0 15%; /* explanation below */
  padding: 25px;
  @media (--viewport-sm-only) {
    flex: 1 0 25%;
  }
  @media (--viewport-md-only) {
    flex: 1 0 25%;
  }
}

.wideItem {
  flex: 1 0 30%;
  padding: 25px;
  @media (--viewport-sm-only) {
    flex: 1 0 25%;
  }
  @media (--viewport-md-only) {
    flex: 1 0 25%;
  }
}
