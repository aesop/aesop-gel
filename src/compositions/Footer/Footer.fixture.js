export default {
  blocks: [
    {
      heading: 'Orders & Support',
      links: [
        {
          text: 'Contact us',
          id: 'contact-us',
          url: 'https://www.aesop.com/au/r/contact-us',
          openInANewWindow: true,
        },
        {
          text: 'Delivery and returns',
          id: 'delivery-and-returns',
          url: 'https://www.aesop.com/au/r/shipping/',
          openInANewWindow: true,
        },
        {
          text: 'Click And Collect',
          id: 'click-and-collect',
          url: 'https://www.aesop.com/au/r/contact-us',
          openInANewWindow: true,
        },
        {
          text: 'Track Your Orders',
          id: 'track-your-orders',
          url: 'https://www.aesop.com/au/r/shipping/',
          openInANewWindow: false,
        },
      ],
    },
    {
      heading: 'Services',
      links: [
        {
          text: 'Video Consultation',
          id: 'video-consultation',
          url: 'https://www.aesop.com/au/r/contact-us',
          openInANewWindow: false,
        },
        {
          text: 'Live Assistance',
          id: 'live-assistance',
          url: 'https://www.aesop.com/au/r/shipping/',
          openInANewWindow: false,
        },
        {
          text: 'Call And Collect',
          id: 'call-and-collect',
          url: 'https://www.aesop.com/au/r/contact-us',
          openInANewWindow: false,
        },
        {
          text: 'Facial Appointment',
          id: 'facial-appointment',
          url: 'https://www.aesop.com/au/r/shipping/',
          openInANewWindow: false,
        },
        {
          text: 'Corporate Gifts',
          id: 'corporate-gifts',
          url: 'https://www.aesop.com/au/r/shipping/',
          openInANewWindow: false,
        },
      ],
    },
    {
      heading: 'Shipping to',
      copy: 'TODO: Shipping To Component',
    },
    {
      heading: 'Acknowledgment of Country',
      copy:
        'Aesop acknowledges the Traditional Owners of the lands on which we live and work, and pays deepest respects to Elders past, present and emerging. We also acknowledge that sovereignty of these lands has never been ceded.',
    },
    {
      heading: 'Sustainability',
      copy:
        'Our approach to sustainability is guided by where we can have the greatest social and environmental impact.',
    },
    {
      heading: 'About',
      links: [
        {
          text: 'About Us',
          id: 'about-us',
          url: 'https://www.aesop.com/au/r/contact-us',
          openInANewWindow: false,
        },
        {
          text: 'Career',
          id: 'career',
          url: 'https://www.aesop.com/au/r/shipping/',
          openInANewWindow: false,
        },
        {
          text: 'Press',
          id: 'press',
          url: 'https://www.aesop.com/au/r/contact-us',
          openInANewWindow: false,
        },
        {
          text: 'Policy',
          id: 'policy',
          url: 'https://www.aesop.com/au/r/shipping/',
          openInANewWindow: false,
        },
        {
          text: 'Privacy Policy',
          id: 'privacy-policy',
          url: 'https://www.aesop.com/au/r/shipping/',
          openInANewWindow: false,
        },
        {
          text: 'Accessibility',
          id: 'accessibility',
          url: 'https://www.aesop.com/au/r/shipping/',
          openInANewWindow: false,
        },
        {
          text: 'Terms and Conditions',
          id: 'terms-and-conditions',
          url: 'https://www.aesop.com/au/r/shipping/',
          openInANewWindow: false,
        },
      ],
    },
    {
      heading: 'Social Media',
      isVisibleOnTabletAndMobile: false,
      links: [
        {
          text: 'Instagram',
          id: 'instagram',
          url: 'https://www.aesop.com/au/r/contact-us',
          openInANewWindow: true,
        },
        {
          text: 'Twitter',
          id: 'twitter',
          url: 'https://www.aesop.com/au/r/contact-us',
          openInANewWindow: true,
        },
        {
          text: 'KaKao Plus Friends',
          id: 'kakao',
          url: 'https://www.aesop.com/au/r/contact-us',
          openInANewWindow: true,
        },
        {
          text: 'Naver Post',
          id: 'naver-post',
          url: 'https://www.aesop.com/au/r/contact-us',
          openInANewWindow: true,
        },
      ],
    },
  ],
  iconLinks: [
    {
      id: 'Instagram',
      altText: 'Instagram',
      icon: {
        altText: 'instagram',
        large: './assets/images/IconLink/instagram-light.png',
        medium: './assets/images/IconLink/instagram-light.png',
        small: './assets/images/IconLink/instagram-light.png',
      },
      target: 'https://about.instagram.com/',
      hasTargetInNewWindow: true,
    },
    {
      id: 'Twitter',
      altText: 'Twitter',
      icon: {
        altText: 'Wwitter',
        large: './assets/images/IconLink/twitter-light.png',
        medium: './assets/images/IconLink/twitter-light.png',
        small: './assets/images/IconLink/twitter-light.png',
      },
      target: 'https://twitter.com/home?lang=en',
      hasTargetInNewWindow: true,
    },
    {
      id: 'LinkedIn',
      altText: 'LinkedIn',
      icon: {
        altText: 'linkedin',
        large: './assets/images/IconLink/linkedin-light.png',
        medium: './assets/images/IconLink/linkedin-light.png',
        small: './assets/images/IconLink/linkedin-light.png',
      },
      target: 'https://www.linkedin.com/home/?originalSubdomain=au',
      hasTargetInNewWindow: true,
    },
  ],
};
