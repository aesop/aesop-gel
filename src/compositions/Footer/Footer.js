import React from 'react';
import PropTypes from 'prop-types';
import cx from 'classnames';
import FooterBlock from '~/components/FooterBlock';
import NotificationModal from '~/components/NotificationModal';
import NewsLetterSignUp from '../NewsLetterSignUp';
import styles from './Footer.css';
import { NotificationContextProvider } from '~/contexts';

const Footer = ({
  blocks,
  className,
  consentErrorMsg,
  errorMessage,
  notificationMessage,
  showTermsConditionsTextBox,
  subscriptionMessage,
  termsAndCondition,
  termsMessage,
  onClick,
}) => {
  const baseClassSet = cx(styles.base, className);
  const blockClassSet = cx(styles.block);
  const bottomClasset = cx(styles.bottom);

  return (
    <NotificationContextProvider>
      <div className={baseClassSet}>
        <NotificationModal
          backgroundColor="#d5d5cc"
          notificationMessage={notificationMessage}
        />
        <div className={blockClassSet}>
          <div className={styles.signup}>
            <NewsLetterSignUp
              consentErrorMsg={consentErrorMsg}
              errorMessage={errorMessage}
              onClick={onClick}
              showTermsConditionsTextBox={showTermsConditionsTextBox}
              subscriptionMessage={subscriptionMessage}
              termsAndCondition={termsAndCondition}
              termsMessage={termsMessage}
              theme="light"
            />
          </div>
          {blocks &&
            blocks.map((block, index) => (
              <FooterBlock
                className={
                  (index + 1) % 4 === 0 ? styles.wideItem : styles.item
                }
                copy={block.copy}
                heading={block.heading}
                isVisibleOnTabletAndMobile={block.isVisibleOnTabletAndMobile}
                key={index}
                links={block.links}
                theme="light"
              />
            ))}
        </div>
        <div className={bottomClasset}>&copy; Aesop</div>
      </div>
    </NotificationContextProvider>
  );
};

Footer.propTypes = {
  blocks: PropTypes.array,
  className: PropTypes.string,
  consentErrorMsg: PropTypes.string,
  errorMessage: PropTypes.string,
  notificationMessage: PropTypes.string,
  showTermsConditionsTextBox: PropTypes.bool,
  subscriptionMessage: PropTypes.string,
  termsAndCondition: PropTypes.string,
  termsMessage: PropTypes.string,
  onClick: PropTypes.func,
};

Footer.defaultProps = {
  blocks: undefined,
  className: undefined,
  consentErrorMsg: undefined,
  errorMessage: undefined,
  notificationMessage: undefined,
  showTermsConditionsTextBox: undefined,
  subscriptionMessage: undefined,
  termsAndCondition: undefined,
  termsMessage: undefined,
  onClick: undefined,
};

export default Footer;
