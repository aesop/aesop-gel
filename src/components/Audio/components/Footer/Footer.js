import React from 'react';
import PropTypes from 'prop-types';
import cx from 'classnames';
import moment from 'moment';
import { HYPERLINK_STYLE_TYPES } from '~/constants';
import { Hyperlink } from '~/components/Hyperlink';
import { Icon } from '~/components/Icon';
import { PausePlayButton } from '~/components/Audio/components/PausePlayButton';
import { SeekBackwardButton } from '~/components/Audio/components/SeekBackwardButton';
import { SeekForwardButton } from '~/components/Audio/components/SeekForwardButton';
import styles from './Footer.module.css';

const Footer = React.memo(
  ({
    audioUrl,
    copy,
    duration,
    isLoading,
    isPlaying,
    onSeekBackwardButtonClick,
    onSeekForwardButtonClick,
    onPlayPauseButtonClick,
    progress,
    progressColor,
  }) => {
    return (
      <footer className={cx(styles.base, styles[progressColor])}>
        <time className={cx(styles.time, { [styles.disabled]: isLoading })}>
          {moment.utc(progress * 1000).format('mm:ss')} /{' '}
          {moment.utc(duration * 1000).format('mm:ss')}
        </time>
        <div className={styles.controls}>
          <SeekForwardButton
            copy={{
              title: copy.seekBackwardTitle,
            }}
            isLoading={isLoading}
            onClick={onSeekForwardButtonClick}
            progressColor={progressColor}
          />
          <PausePlayButton
            copy={{
              loading: copy.loading,
              pause: copy.pause,
              play: copy.play,
            }}
            isLoading={isLoading}
            isPlaying={isPlaying}
            onClick={onPlayPauseButtonClick}
            progressColor={progressColor}
          />
          <SeekBackwardButton
            copy={{
              title: copy.seekBackwardTitle,
            }}
            isLoading={isLoading}
            onClick={onSeekBackwardButtonClick}
            progressColor={progressColor}
          />
        </div>
        <div className={styles.download}>
          <Hyperlink
            className={styles.downloadButton}
            hasTargetInNewWindow={true}
            isDownload={true}
            style={HYPERLINK_STYLE_TYPES.EXTERNAL_NO_ICON_TEXT_LINK}
            title={copy.downloadTitle}
            url={audioUrl}
          >
            {`${copy.downloadLabel} `}
            <Icon
              className={styles.downloadButtonIcon}
              height={13}
              name="download"
              width={13}
            />
          </Hyperlink>
        </div>
      </footer>
    );
  },
);

Footer.propTypes = {
  audioUrl: PropTypes.string,
  copy: PropTypes.shape({
    downloadTitle: PropTypes.string,
    downloadLabel: PropTypes.string,
    loading: PropTypes.string,
    pause: PropTypes.string,
    play: PropTypes.string,
    seekBackwardTitle: PropTypes.string,
    seekForwardTitle: PropTypes.string,
  }),
  duration: PropTypes.number,
  isLoading: PropTypes.bool,
  isPlaying: PropTypes.bool,
  onSeekBackwardButtonClick: PropTypes.func,
  onSeekForwardButtonClick: PropTypes.func,
  onPlayPauseButtonClick: PropTypes.func,
  progress: PropTypes.number,
  progressColor: PropTypes.oneOf(['orange', 'green', 'blue']),
};

Footer.defaultProps = {
  audioUrl: undefined,
  copy: {
    downloadTitle: undefined,
    downloadLabel: undefined,
    loading: undefined,
    pause: undefined,
    play: undefined,
    seekBackwardTitle: undefined,
    seekForwardTitle: undefined,
  },
  duration: undefined,
  isLoading: undefined,
  isPlaying: undefined,
  onSeekBackwardButtonClick: undefined,
  onSeekForwardButtonClick: undefined,
  onPlayPauseButtonClick: undefined,
  progress: undefined,
  progressColor: 'orange',
};

export { Footer };
