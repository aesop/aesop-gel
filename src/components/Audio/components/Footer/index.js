/* istanbul ignore file */
import Footer from './Footer';

export default Footer;
