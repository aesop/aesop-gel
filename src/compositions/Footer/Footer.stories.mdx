import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
import * as knobs from '@storybook/addon-knobs';
import Podium from '~/components/Podium';
import NotificationModal from '~/components/NotificationModal';
import NotificationModalFixture from '~/components/NotificationModal/NotificationModal.fixture';
import Footer from './Footer';
import FooterFixture from './Footer.fixture'
import NewsLetterSignUpFixture from '../NewsLetterSignUp/NewsLetterSignUp.fixture'
import newsLetterSignUpOnClick from '../NewsLetterSignUp/__mocks__/NewsLetterSignUp.onClick';


# Footer

<Meta title="Compositions/Footer" component={Footer} />

<Preview>
  <Story name="Base component">
       <Footer 
        blocks={FooterFixture.blocks}
        consentErrorMsg={NewsLetterSignUpFixture.consentErrorMsg}
        iconLinks={FooterFixture.iconLinks}
        errorMessage={NewsLetterSignUpFixture.errorMessage}
        onClick={newsLetterSignUpOnClick}
        showTermsConditionsTextBox={knobs.boolean('showTermsConditionsTextBox', true, 'Presentation')}
        subscriptionMessage={NewsLetterSignUpFixture.subscriptionMessage}
        termsAndCondition={NewsLetterSignUpFixture.termsAndCondition}
        termsMessage={NewsLetterSignUpFixture.termsMessage}
        notificationMessage={NotificationModalFixture.notificationMessage}
       />
  </Story>
</Preview>

## API

<Props of={ContentHubArticle} />

