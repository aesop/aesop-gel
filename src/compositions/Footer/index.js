/* istanbul ignore file */
import { Footer } from './Footer';

export { Footer };
