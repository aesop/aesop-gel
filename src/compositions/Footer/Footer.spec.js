import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from './Footer';
import { FooterFixture } from './Footer.fixture';
import { NewsletterSignupFixture } from '../NewsletterSignup/NewsletterSignup.fixture';
import { NotificationModalFixture } from '~/components/NotificationModal/NotificationModal.fixture';

describe('<Footer />', () => {
  it('should be defined', () => {
    expect(Footer).toBeDefined();
  });

  it('should render base component correctly', () => {
    const { container } = render(
      <Footer
        blocks={FooterFixture.blocks}
        consentErrorMsg={NewsletterSignupFixture.consentErrorMsg}
        errorMessage={NewsletterSignupFixture.errorMessage}
        notificationMessage={NotificationModalFixture.notificationMessage}
        showTermsConditionsTextBox={true}
        subscriptionMessage={NewsletterSignupFixture.subscriptionMessage}
        termsAndCondition={NewsletterSignupFixture.termsAndCondition}
        termsMessage={NewsletterSignupFixture.termsMessage}
      />,
    );

    expect(container).toMatchSnapshot();
  });
});
