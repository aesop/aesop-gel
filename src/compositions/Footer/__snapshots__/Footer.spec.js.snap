// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders base component correctly 1`] = `
<div
  className="base"
>
  <div
    className="base isNotVisible"
    style={
      Object {
        "backgroundColor": "#d5d5cc",
      }
    }
  >
    <span
      className="notificationMessage"
    >
      Sincere thanks for subscribing. We are always pleased to share ourpassionate interests with like - minded individuals.
    </span>
    <button
      className="closeButton"
      data-ref="close"
      onClick={[Function]}
      tabIndex={-1}
    >
      <svg
        className="base close dark"
        data-ref=""
        focusable="false"
        height={15}
        role="img"
        style={
          Object {
            "height": "15px",
            "width": "15px",
          }
        }
        tabIndex={-1}
        viewBox="0 0 50 50"
        width={15}
      >
        <g>
          <polygon
            className="close"
            points="50,5 45,0 25,20 5,0 0,5 20,25 0,45 5,50 25,30 45,50 50,45 30,25"
          />
        </g>
      </svg>
    </button>
  </div>
  <div
    className="block"
  >
    <div
      className="signup"
    >
      <div
        className="base"
      >
        <form
          onSubmit={[Function]}
        >
          <div
            className="inputField light"
          >
            <div
              className="footerNewsletterInputWrapper"
            >
              <div
                className="formText"
              >
                <label
                  htmlFor="newsletter-email"
                >
                  <input
                    className="formTextInput light footerNewsLetter"
                    disabled={false}
                    id="newsletter-email"
                    name="newsletter-email"
                    noValidate={true}
                    onChange={[Function]}
                    placeholder=""
                    type="text"
                  />
                  <span
                    className="formTextLabel light"
                  >
                    Email Address
                  </span>
                </label>
              </div>
            </div>
            <button
              className="button light"
              disabled={true}
            >
              <svg
                className="base rightArrow light signUpIcon light"
                data-ref=""
                focusable="false"
                height={12}
                role="img"
                style={
                  Object {
                    "height": "12px",
                    "width": "12px",
                  }
                }
                tabIndex={-1}
                viewBox="0 0 50 50"
                width={12}
              >
                <g>
                  <path
                    d="M30.1,5.3L50,25.1L30.1,45h-6.6l18-17.6H0v-4.8h41.5l-18-17.6h6.6V5.3z"
                  />
                </g>
              </svg>
            </button>
          </div>
          <section>
            <label
              className="base"
              htmlFor="subscription"
            >
              <input
                className="checkBox light"
                disabled={false}
                id="subscription"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                className="content light"
              >
                Subscribe to communications about Aesop products, services, stores, events and matters of cultural interest.
              </span>
            </label>
            <label
              className="base"
              htmlFor="terms"
            >
              <input
                className="checkBox light"
                disabled={false}
                id="terms"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                className="content light"
              >
                I accept the Terms & Conditions 
              </span>
            </label>
            <label
              htmlFor="term-condition"
            >
              <div
                className="base light"
              >
                <textarea
                  className="textarea light"
                  disabled={true}
                  id="term-condition"
                  rows={3}
                  value="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque quis sem quis sem bibendum aliquet non nec sapien. Suspendisse felis dui, gravida non ligula id, tempor pharetra sapien. Vestibulum porttitor felis vitae mollis porta. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque quis sem quis sem bibendum aliquet non nec sapien. Suspendisse felis dui, gravida non ligula id, tempor pharetra sapien. Vestibulum porttitor felis vitae mollis porta. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque quis sem quis sem bibendum aliquet non nec sapien. Suspendisse felis dui, gravida non ligula id, tempor pharetra sapien. Vestibulum porttitor felis vitae mollis porta. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque quis sem quis sem bibendum aliquet non nec sapien. Suspendisse felis dui, gravida non ligula id, tempor pharetra sapien. Vestibulum porttitor felis vitae mollis porta."
                />
              </div>
            </label>
          </section>
        </form>
      </div>
    </div>
    <section
      className="base light item"
    >
      <h5
        className="heading"
      >
        Orders & Support
      </h5>
      <div
        className="divider light"
      />
      <ul
        className="list"
      >
        <li>
          <a
            className="base hasIcon hasExternalIcon left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_contact-us"
            href="https://www.aesop.com/au/r/contact-us"
            target="_self"
            title="Contact us"
          >
            Contact us
             
            <i
              aria-hidden="true"
              className="icon"
            >
              <svg
                className="base rightUpArrow light"
                data-ref=""
                focusable="false"
                height={12}
                role="img"
                style={
                  Object {
                    "height": "12px",
                    "width": "12px",
                  }
                }
                tabIndex={-1}
                viewBox="0 0 50 50"
                width={12}
              >
                <g>
                  <path
                    d="M21.8,0.2l28.1-0.1v28.1l-4.5,0l0.1-20.5L16.2,37.1l-3.4-3.4L42.1,4.4L21.8,4.4L21.8,0.2L21.8,0.2z"
                  />
                </g>
              </svg>
            </i>
          </a>
        </li>
        <li>
          <a
            className="base hasIcon hasExternalIcon left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_delivery-and-returns"
            href="https://www.aesop.com/au/r/shipping/"
            target="_self"
            title="Delivery and returns"
          >
            Delivery and returns
             
            <i
              aria-hidden="true"
              className="icon"
            >
              <svg
                className="base rightUpArrow light"
                data-ref=""
                focusable="false"
                height={12}
                role="img"
                style={
                  Object {
                    "height": "12px",
                    "width": "12px",
                  }
                }
                tabIndex={-1}
                viewBox="0 0 50 50"
                width={12}
              >
                <g>
                  <path
                    d="M21.8,0.2l28.1-0.1v28.1l-4.5,0l0.1-20.5L16.2,37.1l-3.4-3.4L42.1,4.4L21.8,4.4L21.8,0.2L21.8,0.2z"
                  />
                </g>
              </svg>
            </i>
          </a>
        </li>
        <li>
          <a
            className="base hasIcon hasExternalIcon left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_click-and-collect"
            href="https://www.aesop.com/au/r/contact-us"
            target="_self"
            title="Click And Collect"
          >
            Click And Collect
             
            <i
              aria-hidden="true"
              className="icon"
            >
              <svg
                className="base rightUpArrow light"
                data-ref=""
                focusable="false"
                height={12}
                role="img"
                style={
                  Object {
                    "height": "12px",
                    "width": "12px",
                  }
                }
                tabIndex={-1}
                viewBox="0 0 50 50"
                width={12}
              >
                <g>
                  <path
                    d="M21.8,0.2l28.1-0.1v28.1l-4.5,0l0.1-20.5L16.2,37.1l-3.4-3.4L42.1,4.4L21.8,4.4L21.8,0.2L21.8,0.2z"
                  />
                </g>
              </svg>
            </i>
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_track-your-orders"
            href="https://www.aesop.com/au/r/shipping/"
            target="_self"
            title="Track Your Orders"
          >
            Track Your Orders
          </a>
        </li>
      </ul>
    </section>
    <section
      className="base light item"
    >
      <h5
        className="heading"
      >
        Services
      </h5>
      <div
        className="divider light"
      />
      <ul
        className="list"
      >
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_video-consultation"
            href="https://www.aesop.com/au/r/contact-us"
            target="_self"
            title="Video Consultation"
          >
            Video Consultation
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_live-assistance"
            href="https://www.aesop.com/au/r/shipping/"
            target="_self"
            title="Live Assistance"
          >
            Live Assistance
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_call-and-collect"
            href="https://www.aesop.com/au/r/contact-us"
            target="_self"
            title="Call And Collect"
          >
            Call And Collect
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_facial-appointment"
            href="https://www.aesop.com/au/r/shipping/"
            target="_self"
            title="Facial Appointment"
          >
            Facial Appointment
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_corporate-gifts"
            href="https://www.aesop.com/au/r/shipping/"
            target="_self"
            title="Corporate Gifts"
          >
            Corporate Gifts
          </a>
        </li>
      </ul>
    </section>
    <section
      className="base light item"
    >
      <h5
        className="heading"
      >
        Shipping to
      </h5>
      <div
        className="divider light"
      />
      <ul
        className="list"
      />
      <div
        className="listItem"
      >
        TODO: Shipping To Component
      </div>
    </section>
    <section
      className="base light wideItem item"
    >
      <h5
        className="heading"
      >
        Acknowledgment of Country
      </h5>
      <div
        className="divider light"
      />
      <ul
        className="list"
      />
      <div
        className="listItem"
      >
        Aesop acknowledges the Traditional Owners of the lands on which we live and work, and pays deepest respects to Elders past, present and emerging. We also acknowledge that sovereignty of these lands has never been ceded.
      </div>
    </section>
    <section
      className="base light item"
    >
      <h5
        className="heading"
      >
        Sustainability
      </h5>
      <div
        className="divider light"
      />
      <ul
        className="list"
      />
      <div
        className="listItem"
      >
        Our approach to sustainability is guided by where we can have the greatest social and environmental impact.
      </div>
    </section>
    <section
      className="base light item"
    >
      <h5
        className="heading"
      >
        About
      </h5>
      <div
        className="divider light"
      />
      <ul
        className="list"
      >
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_about-us"
            href="https://www.aesop.com/au/r/contact-us"
            target="_self"
            title="About Us"
          >
            About Us
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_career"
            href="https://www.aesop.com/au/r/shipping/"
            target="_self"
            title="Career"
          >
            Career
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_press"
            href="https://www.aesop.com/au/r/contact-us"
            target="_self"
            title="Press"
          >
            Press
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_policy"
            href="https://www.aesop.com/au/r/shipping/"
            target="_self"
            title="Policy"
          >
            Policy
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_privacy-policy"
            href="https://www.aesop.com/au/r/shipping/"
            target="_self"
            title="Privacy Policy"
          >
            Privacy Policy
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_accessibility"
            href="https://www.aesop.com/au/r/shipping/"
            target="_self"
            title="Accessibility"
          >
            Accessibility
          </a>
        </li>
        <li>
          <a
            className="base left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_terms-and-conditions"
            href="https://www.aesop.com/au/r/shipping/"
            target="_self"
            title="Terms and Conditions"
          >
            Terms and Conditions
          </a>
        </li>
      </ul>
    </section>
    <section
      className="base light item"
    >
      <h5
        className="heading"
      >
        Social Media
      </h5>
      <div
        className="divider light"
      />
      <ul
        className="list"
      >
        <li>
          <a
            className="base hasIcon hasExternalIcon left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_instagram"
            href="https://www.aesop.com/au/r/contact-us"
            target="_self"
            title="Instagram"
          >
            Instagram
             
            <i
              aria-hidden="true"
              className="icon"
            >
              <svg
                className="base rightUpArrow light"
                data-ref=""
                focusable="false"
                height={12}
                role="img"
                style={
                  Object {
                    "height": "12px",
                    "width": "12px",
                  }
                }
                tabIndex={-1}
                viewBox="0 0 50 50"
                width={12}
              >
                <g>
                  <path
                    d="M21.8,0.2l28.1-0.1v28.1l-4.5,0l0.1-20.5L16.2,37.1l-3.4-3.4L42.1,4.4L21.8,4.4L21.8,0.2L21.8,0.2z"
                  />
                </g>
              </svg>
            </i>
          </a>
        </li>
        <li>
          <a
            className="base hasIcon hasExternalIcon left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_twitter"
            href="https://www.aesop.com/au/r/contact-us"
            target="_self"
            title="Twitter"
          >
            Twitter
             
            <i
              aria-hidden="true"
              className="icon"
            >
              <svg
                className="base rightUpArrow light"
                data-ref=""
                focusable="false"
                height={12}
                role="img"
                style={
                  Object {
                    "height": "12px",
                    "width": "12px",
                  }
                }
                tabIndex={-1}
                viewBox="0 0 50 50"
                width={12}
              >
                <g>
                  <path
                    d="M21.8,0.2l28.1-0.1v28.1l-4.5,0l0.1-20.5L16.2,37.1l-3.4-3.4L42.1,4.4L21.8,4.4L21.8,0.2L21.8,0.2z"
                  />
                </g>
              </svg>
            </i>
          </a>
        </li>
        <li>
          <a
            className="base hasIcon hasExternalIcon left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_kakao"
            href="https://www.aesop.com/au/r/contact-us"
            target="_self"
            title="KaKao Plus Friends"
          >
            KaKao Plus Friends
             
            <i
              aria-hidden="true"
              className="icon"
            >
              <svg
                className="base rightUpArrow light"
                data-ref=""
                focusable="false"
                height={12}
                role="img"
                style={
                  Object {
                    "height": "12px",
                    "width": "12px",
                  }
                }
                tabIndex={-1}
                viewBox="0 0 50 50"
                width={12}
              >
                <g>
                  <path
                    d="M21.8,0.2l28.1-0.1v28.1l-4.5,0l0.1-20.5L16.2,37.1l-3.4-3.4L42.1,4.4L21.8,4.4L21.8,0.2L21.8,0.2z"
                  />
                </g>
              </svg>
            </i>
          </a>
        </li>
        <li>
          <a
            className="base hasIcon hasExternalIcon left light listItem"
            data-test-ref="DATA_TEST_REF_FOOTER_BLOCK_naver-post"
            href="https://www.aesop.com/au/r/contact-us"
            target="_self"
            title="Naver Post"
          >
            Naver Post
             
            <i
              aria-hidden="true"
              className="icon"
            >
              <svg
                className="base rightUpArrow light"
                data-ref=""
                focusable="false"
                height={12}
                role="img"
                style={
                  Object {
                    "height": "12px",
                    "width": "12px",
                  }
                }
                tabIndex={-1}
                viewBox="0 0 50 50"
                width={12}
              >
                <g>
                  <path
                    d="M21.8,0.2l28.1-0.1v28.1l-4.5,0l0.1-20.5L16.2,37.1l-3.4-3.4L42.1,4.4L21.8,4.4L21.8,0.2L21.8,0.2z"
                  />
                </g>
              </svg>
            </i>
          </a>
        </li>
      </ul>
    </section>
  </div>
  <div
    className="bottom"
  >
    © Aesop
  </div>
</div>
`;
